### DISCUSSION TITLE
Make the "seconds" field optionnal for the clock Widget

### DISCUSSION THREAD

**Discussion by Antiz96** [Lissy93/dashy#644] [Ideas]
Hi !

I love the new widgets implementation, thank you for this awesome work !

I use the "clock" widget (among others) but I was wondering if it could be possible to set the "seconds" field as optional somehow ?
Maybe with something like "showSeconds: true/false" ?

Anyway, thanks a lot once again !

Robin

**Comment by lissy93** [comment]
Heya Robin,
Yeah this should be easy enough, and makes sense to add.
I can include this in the next PR, probably this week :)

Otherwise, if anyone's looking for an easy first PR, this would make a good task. In the [`Clock.vue`](https://github.com/lissy93/dashy/blob/master/src/components/Widgets/Clock.vue) file:
1) Under the computed properties, get the users pref for `options.hideSeconds`. `showSeconds() { return !this.options.hideSeconds; },`
2) Then under [`setTime() {...}`](https://github.com/lissy93/dashy/blob/master/src/components/Widgets/Clock.vue#L51) method, just replace `second: 'numeric',` with `...(this.showSeconds && { second: 'numeric' }),`
3) Then just add the new option to the widget docs under Clock section, and check everything looks good

**Reply by Antiz96** [reply]
Hi Lissy,

Thanks for your quick answer !

I would love to submit the PR myself but I'm not really comfortable with vue. 
Even tho the changes look fairly easy, I'm afraid to make some kind of mistakes anyway... I should probably let someone else handle that I guess !

Anyway, I'm glad this new feature doesn't require much. 
Thank you for considering it :)

Robin
